hdfs/src/test/java/io/ddf/hdfs/HDFSDDFManagerTests.java

```diff
@@ -0,0 +1,113 @@
+package io.ddf.hdfs;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+import io.ddf.datasource.DataFormat;
+import io.ddf.exception.DDFException;
+
+
+public class HDFSDDFManagerTests {
```

Comment:
Should have tests for get data format for these cases:
- unsupported data format
- have more than 2 formats